nomination/templates/nomination/url_add.html

```diff
@@ -1,222 +1,214 @@
+                        <li class="list-group-item">
+                            <h4>Information About You
+                            {% if project.registration_required %}
+                                <span class="label label-info">Required</span>
+                            {% endif %}
+                            </h4>
+                            <div class="form-group">
+                                {% if form_errors.nominator_name %}
+                                    <div class="alert alert-danger" role="alert">{{ form_errors.nominator_name|striptags }}</div>
+                                {% endif %}
+                                <label for="your-name-value">Your Name:</label>
+                                {{ form.nominator_name|add_class:"form-control" }}
+                                <p class="help-block">Please provide your name</p>
+                            </div>
+                            <div class="form-group">
+                                {% if form_errors.nominator_institution %}
+                                    <div class="alert alert-danger" role="alert">{{ form_errors.nominator_institution|striptags }}</div>
+                                {% endif %}
+                                <label for="institution-value">Institution:</label>
+                                {{ form.nominator_institution|add_class:"form-control" }}
+                                <p class="help-block">Please provide your institutional affiliation</p>
+                            </div>
+                            <div class="form-group">
+                                {% if form_errors.nominator_email %}
+                                    <div class="alert alert-danger" role="alert">{{ form_errors.nominator_email|striptags }}</div>
+                                {% endif %}
+                                <label for="email-value">Your E-Mail Address:</label>
+                                {{ form.nominator_email|add_class:"form-control" }}
+                                <p class="help-block">Please provide an email address in order to identify your nominations in the system</p>
+                            </div>
+                            <input type="submit" value="submit" class="btn btn-primary" />
+                        </li>
+                    </form>
+                {% else %}
+                    <br><div class="alert alert-info" role="alert">{{ project.nomination_message }}</div>
                 {% endif %}
-                {{ form.nominator_name }}
-                </div>
-                <div class="instruction">Please provide your name</div>
-
-                <label for="institution-value">Institution:</label>
-                <div class="element-wrapper">
-                {% if form_errors.nominator_institution %}
-                    <span class="required" title="Required Field">{{form_errors.nominator_institution}}</span><br/>
+                {% if json_data %}
```

Comment:
I don't know why this script is inside the list, should probably go in the bottom of body.